app/models/tag.rb

```diff
@@ -61,6 +61,12 @@ def self.valid_categories
     ALLOWED_CATEGORIES
   end
 
+  def self.aliased_name(word)
+    return unless (tag = find_by(name: word.downcase))
```

Comment:
I'm not sure saving one line here is fruitful for readability, I'd split this into:

```ruby
tag = ...
return unless tag
```